src/event-db/src/queries/event/proposal.rs

```diff
@@ -0,0 +1,102 @@
+use crate::{
+    error::Error,
+    types::event::{objective::ObjectiveId, proposal::ProposalSummary},
+    EventDB,
+};
+use async_trait::async_trait;
+use serde::Deserialize;
+
+#[derive(Deserialize)]
+pub enum VoterGroup {
```

Comment:
Nit, it was discussed before, we don't have enums as a representation of some db fields, even if in our service API they are defined as enums. The main intention is not to have a strict validation of these fields in our service, so in case when we want to update or expand these values we will just need to update our API documentation and put new values into the db, and we don't want to update service.
So you should use just a wrapper over `String` type at this place
```
pub struct VoterGroup(pub String);
```
P.S. Here is an example the same field https://github.com/input-output-hk/catalyst-core/blob/0444b08ec66c80ae95cf8bbdbe606d0ed1bd6a06/src/event-db/src/types/event/mod.rs#L422. (if you will introduce a new type, can you pls modify this place too pls 😊)